### ISSUE TITLE
Voices are broken across newlines

### ISSUE THREAD

**Issue by elyisgreat** [alda-lang/alda#228]
I just installed alda 1.0.0-rc15 and noticed something odd about the way voices behave. It seems as though they are broken across newlines. For example, take this code:

```
piano:

V1: o4 c d e f
V2: o4 e f g a

V1: g a b > c
V2: b > c d e
```

In theory this should play identically to

```
piano:

V1: o4 c d e f g a b > c
V2: o4 e f g a b > c d e
```

and this

```
piano "pi1": o4 c d e f
piano "pi2": o4 e f g a

pi1: g a b > c
pi2: b > c d e
```

However instead of playing the voices sequentially, it plays each line all at once. I'm pretty sure this isn't supposed to happen...

**Comment by daveyarwood** [comment]
Oops! That's definitely a bug. In your first example, I think it's treating each invocation of a voice (i.e. V1, then V2, then V1, then V2) as a separate voice, instead of two voices with continuations.

Good catch! I'll work on fixing this ASAP.

**Comment by daveyarwood** [comment]
Fixed in [1.0.0-rc16](https://github.com/alda-lang/alda/releases/tag/1.0.0-rc16).
